drizzle-orm/src/aws-data-api/pg/driver.ts

```diff
@@ -152,7 +152,7 @@ export function drizzle<
 	$client: TClient;
 } {
 	// eslint-disable-next-line no-instanceof/no-instanceof
```

Comment:
Thank you for this PR! I'm patiently awaiting for it to be merged. I'm not yet a contributor to drizzle, but can't this line be omitted now that we remove the use of instanceof on the following line?

Reply:
good point, thank you!